scripts/task_config.yaml

```diff
@@ -0,0 +1,33 @@
+# Grid World
+
+grid_size: 5
```

Comment:
I'm in favor of having the graph defined outside of the yaml which should only contain a list of tasks

Reply:
Yes, for the time being we will use a "grid world" as it is straightforward to implement. I will try to add this function in a future PR.